packages/paymaster/src/utils/HelperFunction.ts

```diff
@@ -0,0 +1,24 @@
+import { UserOperation } from "@biconomy/core-types";
+import { BigNumber } from "ethers";
+
+export const transformUserOP = (userOp: UserOperation): UserOperation => {
```

Comment:
we didn't need this before? 
at which point we will need to transform these now. converting all to hex string

Reply:
I saw we do this exact thing in bundler so kept the same here in bundler also as the types are same